modules/sizeMappingV2.js

```diff
@@ -429,57 +438,56 @@ export function getRelevantMediaTypesForBidder(sizeConfig, activeViewport) {
 
 // sets sizeMappingInternalStore for a given auctionId with relevant adUnit information returned from the call to 'getFilteredMediaTypes' function
 // returns adUnit details object.
-export function getAdUnitDetail(auctionId, adUnit) {
+export function getAdUnitDetail(auctionId, adUnit, labels) {
   // fetch all adUnits for an auction from the sizeMappingInternalStore
   const adUnitsForAuction = sizeMappingInternalStore.getAuctionDetail(auctionId).adUnits;
 
   // check if the adUnit exists already in the sizeMappingInterStore (check for equivalence of 'code' && 'mediaTypes' properties)
   const adUnitDetail = adUnitsForAuction.filter(adUnitDetail => adUnitDetail.adUnitCode === adUnit.code && utils.deepEqual(adUnitDetail.mediaTypes, adUnit.mediaTypes));
 
   if (adUnitDetail.length > 0) {
+    adUnitDetail[0].cacheHits++;
     return adUnitDetail[0];
   } else {
-    const { mediaTypes, sizeBucketToSizeMap, activeViewport, transformedMediaTypes } = internal.getFilteredMediaTypes(adUnit.mediaTypes);
+    const identicalAdUnit = adUnitsForAuction.filter(adUnitDetail => adUnitDetail.adUnitCode === adUnit.code);
+    const adUnitInstance = identicalAdUnit.length > 0 ? typeof identicalAdUnit[0].instance === 'number' ? identicalAdUnit[identicalAdUnit.length - 1].instance + 1 : 1 : 1;
```

Comment:
Are the nested ternary operators needed here? Can this be rewritten as:
```
identicalAdUnit.length > 0 && typeof identicalAdUnit[0].instance === 'number' ? identicalAdUnit[identicalAdUnit.length - 1].instance + 1 : 1;
```

Reply:
@msm0504 Good catch! I've made the necessary change.